hack/tools/go.mod

```diff
@@ -1,13 +1,17 @@
-module sigs.k8s.io/cluster-api/hack/tools
+module sigs.k8s.io/cluster-api-provider-vsphere/hack/tools
 
 go 1.15
 
 require (
 	github.com/golangci/golangci-lint v1.43.0
-	github.com/onsi/ginkgo v1.16.4
+	github.com/google/gofuzz v1.2.0 // indirect
+	github.com/imdario/mergo v0.3.12 // indirect
+	github.com/mattn/go-runewidth v0.0.13 // indirect
+	github.com/onsi/ginkgo v1.16.5
 	github.com/vmware/govmomi v0.23.1
-	k8s.io/code-generator v0.22.2
-	sigs.k8s.io/controller-tools v0.7.0
+	k8s.io/code-generator v0.23.0-alpha.4
+	sigs.k8s.io/cluster-api/hack/tools v0.0.0-20211111175208-4cc2fce2111a
```

Comment:
Curious as to why would this be needed here. Maybe revert

Reply:
The more I think about this, I think it is required to pull in the `conversion-verifier` binary, right?

Reply:
> The more I think about this, I think it is required to pull in the `conversion-verifier` binary, right?

Yes, that's right!